fillers/example/acl_example.py

```diff
@@ -0,0 +1,75 @@
+"""
+Test ACL Transaction Source Code Examples
+"""
+
+from ethereum_test_tools import (
+    AccessList,
+    Account,
+    Environment,
+    StateTest,
+    Transaction,
+    test_from_until,
+)
+
+
+@test_from_until("berlin", "london")
+def test_access_list(fork):
+    """
+    Test type 1 transaction.
+    """
+    env = Environment(
+        coinbase="0x2adc25665018aa1fe0e6bc666dac8fc2697ff9ba",
+        difficulty=0x20000,
+        gas_limit=68719476736,
+        number=16777216,
+        timestamp=4,
+    )
+
+    pre = {
+        "0x000000000000000000000000000000000000aaaa": Account(
+            balance=0x03,
+            code="0x5854505854",
```

Comment:
```suggestion
            code=Op.PC + Op.SLOAD + Op.POP + Op.PC + Op.SLOAD,
```
It's easier to read, and simply requires this import:
```
from ethereum_test_tools import Opcodes as Op
```

Reply:
done